### ISSUE TITLE
Training without moving objects masks

### ISSUE THREAD

**Issue by maliksyria** [Brummi/MonoRec#22]
Hello!
Thanks for the interesting work. Now, I'm in the process of training the model in my own dataset.
The dataset now contains stereo images with their depth image, and also sparse depth images with camera positions generated by DSO as you suggested.
My dataset does not have such masks of moving objects like those used in KITTI example. 
So my question is, is it possible to train the model without masks? and if yes what will be the correct steps to train it without taking into account these masks?
Thanks in advance

**Comment by maliksyria** [comment]
any answer?

**Comment by Brummi** [comment]
Hi!
Thank you for your interest and sorry for the late reply!
You can indeed train the depth module with your data.
For this, please check out the first training step (depth bootstrap).
If you don't have any masks, you should set 
`"return_mvobj_mask": false` in the training config file for the data loaders.
The config file for the first training step is [configs/train/monorec/monorec_depth.json](https://github.com/Brummi/MonoRec/blob/main/configs/train/monorec/monorec_depth.json).
This will stop the dataloader from trying to load the masks.

Best, Felix
